src/datepicker/datepicker-navigation.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+import {NavigationEvent} from './datepicker-view-model';
+
+@Component({
+  selector: 'tbody[ngb-datepicker-navigation]',
+  styles: [`
+    td {
+      text-align: center;
```

Comment:
AFAIK there is a Bootstrap utlity CSS class for this, might be a good idea to use it instead?